src/types.js

```diff
@@ -283,3 +283,12 @@ export type HvComponent<Props> = ComponentType<Props> & {
   localName: LocalName | string,
   namespaceURI: NamespaceURI,
 };
+
+export type HvBehavior = {
+  action: string,
+  callback: (element: Element, onUpdate: HvComponentOnUpdate) => any,
```

Comment:
- Curious about the `any`. What happens to any returned values here? If we don't do anything with them, I think it's best to make this `void` or `Promise<void>`
- I think it might be good to add documentation around the Flow type to explain the effects of each attribute (e.g. what do we do with the return value of `callback`? Is `action` supposed to be the name of the action?)